src/ngx_http_lua_ssl_certby.c

```diff
@@ -1294,4 +1294,119 @@ ngx_http_lua_ffi_set_priv_key(ngx_http_request_t *r,
 }
 
 
+static int
+ngx_ssl_verify_callback(int ok, X509_STORE_CTX *x509_store)
+{
+    /*
+     * we never terminate handshake here and user can later use
+     * $ssl_client_verify to check verification result.
+     *
+     * this is consistent with Nginx behavior.
+     */
+    return 1;
+}
+
+
+int
+ngx_http_lua_ffi_ssl_verify_client(ngx_http_request_t *r, void *ca_certs,
```

Comment:
if arg ca_certs is path of the cert, then the code would be must easizer.  just like ngx_ssl_client_certificate
why not use path ?

Reply:
Since `ssl.set_cert` accepts a pointer rather than a path, I think it should better to keep them consistent. Besides, it provides some flexibility that these certificates do not have to be on disk. For example, in [Kubernetes ingress controller](https://github.com/kubernetes/ingress-nginx) use case, certificates are kept in Nginx memory and are update via a REST API.